fedimint-ui/apps/gateway-ui/src/index.tsx

```diff
@@ -17,16 +17,16 @@ const mintgate = gateway_api ? new MockMintgate() : new MockMintgate();
 const explorer = new BlockstreamExplorer('https://blockstream.info/api/');
 
 const root = ReactDOM.createRoot(
-	document.getElementById('root') as HTMLElement
+  document.getElementById('root') as HTMLElement
 );
 
 root.render(
-	<React.StrictMode>
-		<Fonts />
-		<ChakraProvider theme={theme}>
-			<ApiProvider props={{ mintgate, explorer }}>
-				<Admin />
-			</ApiProvider>
-		</ChakraProvider>
-	</React.StrictMode>
+  <React.StrictMode>
+    <Fonts />
+    <ChakraProvider theme={theme}>
```

Comment:
Should this be using the `<SharedChakraProvider>` like guardian-ui? Or not yet?

Reply:
Correct, it should be. Didn't bother changing it because gateway-ui doesn't consume any of the shared UI elements just yet